### ISSUE TITLE
Exports display error about failing to get the modified time of the cursor images.

### ISSUE THREAD

**Issue by drwhut** [drwhut/tabletop-club#2]
**Describe the bug**
When running an exported version of the game (i.e. not in the editor), an error is displayed when entering the 3D environment about the engine failing to get the modified time of the cursor images, which are used for displaying other player's cursors.

**To Reproduce**
1. Export the game.
2. Run the executable via the terminal.
3. Start a singleplayer game.
4. The error should be displayed in the terminal.

**Expected behavior**
This error should be displayed:
```
ERROR: _get_modified_time: Failed to get modified time for: res://Images/GrabbingCursor.png.
   At: drivers/unix/file_access_unix.cpp:321.
ERROR: _get_modified_time: Failed to get modified time for: res://Images/ArrowCursor.png.
   At: drivers/unix/file_access_unix.cpp:321.
```

**Screenshots**
None.

**Environment**
Linux Mint 20

**Version**
master

**Additional context**
None.

**Comment by drwhut** [comment]
It seems that Godot itself is causing the issue here, according to this open issue: godotengine/godot#25318

**Comment by drwhut** [comment]
This issue was fixed in Godot 3.4.1!
